core/src/consensus.rs

```diff
@@ -599,7 +599,7 @@ impl Tower {
         self.last_vote = new_vote;
     }
 
-    fn record_bank_vote_and_update_lockouts(
```

Comment:
nit: can we reuse `record_vote` instead of changing the visibility on this? looks like that one is already used for tests.

Reply:
good call, changed